docs/spec/nsm-monitor.md

```diff
@@ -0,0 +1,72 @@
+
+Sidecar nsm-monitor
+============================
+
+Specification
+-------------
+nsm-monitor is simple sidecar for monitoring connection events into client's POD. Also, it providing API for creating custom nsm-monitor (see below).  
```

Comment:
Could you say a bit more about why this might be useful?  You hint at it from references below, but a paragraph here might be very educational :)

Reply:
Doc is updated. Please take a look